.travis.yml

```diff
@@ -3,41 +3,63 @@
 osx_image: xcode8.3
 sudo: required
 dist: trusty
-language: c
+
+language: node_js
+node_js:
+  - 8
+
 matrix:
   include:
   - os: osx
   - os: linux
     env: CC=clang CXX=clang++ npm_config_clang=1
     compiler: clang
+
 cache:
   directories:
   - node_modules
   - "$HOME/.electron"
   - "$HOME/.cache"
+
 addons:
   apt:
     packages:
     - libgnome-keyring-dev
     - icnsutils
+    - graphicsmagick
+    - xz-utils
     #- xvfb
+
 before_install:
-- mkdir -p /tmp/git-lfs && curl -L https://github.com/github/git-lfs/releases/download/v1.2.1/git-lfs-$([
-  "$TRAVIS_OS_NAME" == "linux" ] && echo "linux" || echo "darwin")-amd64-1.2.1.tar.gz
-  | tar -xz -C /tmp/git-lfs --strip-components 1 && /tmp/git-lfs/git-lfs pull
-- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get install --no-install-recommends -y icnsutils graphicsmagick xz-utils; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get -qq update ; fi
+  #- if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update ; fi
+
 install:
-#- export DISPLAY=':99.0'
-#- Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
-- nvm install 8
-- curl -o- -L https://yarnpkg.com/install.sh | bash
-- source ~/.bashrc
-- npm install -g xvfb-maybe
-- yarn
+  #- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then export DISPLAY=':99.0' ; fi
+  #- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 & ; fi
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - source ~/.bashrc
+  - npm install -g xvfb-maybe
+
+  - $CC --version
+  - $CXX --version
+  - npm --version
+  - yarn --version
+
+  # TODO: debug
+  - which yarn
+
+  - yarn
+
 script:
-#- xvfb-maybe node_modules/.bin/karma start test/unit/karma.conf.js
-#- yarn run pack && xvfb-maybe node_modules/.bin/mocha test/e2e
```

Comment:
@Jocs Do you still need that?

Reply:
i don't need it recently, is this config faster than before?

Reply:
I don't think so and the OSX build still does not start. I think it's a problem to their infrastructure because the OSX system is "queued".

Reply:
To check that look here: https://www.traviscistatus.com/
And indeed they had earlier build problems with macosx.

Reply:
I pushed marktext to a private repo. and the OSX build starts immediately. Maybe problems with `travis-ci.org`.

@Jocs Can you please cancel build `#259` on travis-ci.

Reply:
@fxha cancelled. 😴

Reply:
Now they are doing a DB update.....